continuous-integration/readme.md

```diff
@@ -70,6 +70,7 @@ An automated build should encompass the following principles:
   - We encourage maintaining a consistent developer experience for all team members. There should be a central automated manifest / process that streamlines the installation and setup of any software dependencies. This way developers can replicate the same build environment locally as the one running on a CI server.
```

Comment:
This may already have been done, and I just don't see it in this pull request, but is it worth adding an example of how to have consistent local/CI experience with dev containers?

Reply:
Agree that it's worthwhile adding an reference example in this section.... I added a blog post from @stuartleeks as a reference to help elaborate around dev container benefits and best practices.

Reply:
That looks perfect.  After our chat yesterday, I was also wondering if it would be a good idea to list some guidelines for contributing dev containers from CSE.  Can be handled in a future pull request, I just didn't want to lose the train of thought.  @erikschlegel